### PR TITLE
Pass list of URLs to cache flusher instead of one-string-at-a-time.

### PR DESCRIPTION
**Pull request by mparent61**
Cleaned up cache flushing support:
- Allows batch flush operations
- Adds reference cache flush method (pointing to internal CacheMaster flush service).

This is a breaking change so bumping to v2.1.

Companion to https://github.com/theonion/cachemaster/pull/6

@MichaelButkovic @benghaziboy

### CODE REVIEW

File: tests/test_image_model.py [hunk 1]
```diff
@@ -35,37 +35,10 @@ def make_some_crops(image, settings):
 
 @pytest.mark.django_db
 @pytest.mark.usefixtures("clean_image_root")
-def test_image_clear_crops_save_disabled(image, settings):
+@pytest.mark.parametrize('save_crops', [True, False])  # Test setting enabled + disabled
```
Comment (mparent61) [inline tests/test_image_model.py]: I merged 2 nearly-identical tests into one, using py.test's "parameterized tests" -- very cool stuff

**Comment by mparent61** [comment]
@MichaelButkovic @benghaziboy   

This is ready for review

File: betty/contrib/cacheflush/cachemaster.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+from betty.conf.app import settings
+
+logger = __import__('logging').getLogger(__name__)
+
+
+def flush(paths):
+
+    if not hasattr(settings, 'CACHEMASTER_URLS') or not isinstance(settings.CACHEMASTER_URLS, list):
+        logger.error('Invalid/missing setting: "CACHEMASTER_URLS" - list of string URLS')
```
Comment (benghaziboy) [inline betty/contrib/cacheflush/cachemaster.py]: any reason to not raise here? Is this being called async?

**Comment by benghaziboy** [comment]
LGTM

File: betty/contrib/cacheflush/cachemaster.py (hunk 2 above)
Comment (mparent61) [inline betty/contrib/cacheflush/cachemaster.py]: Yeah, better to raise

**Pull request merged**